phoenix4-connectors-assembly/pom.xml

```diff
@@ -0,0 +1,229 @@
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <skipAssembly>${skip-phoenix4}</skipAssembly>
+                            <descriptors>
+                                <descriptor>src/build/package-phoenix4-connectors-to-tar-all.xml</descriptor>
+                            </descriptors>
+                            <finalName>phoenix4-connectors-${project.version}</finalName>
+                            <tarLongFileMode>posix</tarLongFileMode>
+                            <appendAssemblyId>false</appendAssemblyId>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>exec-maven-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <executions>
+                    <execution>
+                        <id>flume without version</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>ln</executable>
+                            <workingDirectory>${project.basedir}/../phoenix-flume-base/phoenix4-flume/target</workingDirectory>
+                            <arguments>
+                                <argument>-fnsv</argument>
+                                <argument>
+                                    phoenix4-flume-${project.version}.jar
+                                </argument>
+                                <argument>
+                                    <!-- We are overwriting the unshaded server JAR, but we don't care -->
+                                    phoenix-flume.jar
+                                </argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>hive without version</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>ln</executable>
+                            <workingDirectory>${project.basedir}/../phoenix-hive-base/phoenix4-hive/target</workingDirectory>
+                            <arguments>
+                                <argument>-fnsv</argument>
+                                <argument>
+                                    phoenix4-hive-${project.version}.jar
+                                </argument>
+                                <argument>
+                                    <!-- We are overwriting the unshaded server JAR, but we don't care -->
+                                    phoenix-hive.jar
```

Comment:
I am not sure if removing phoenix4 / phoenix5 from the name is a good idea. I lean towards keeping that part. (same applies to every symlink)